meetupselector/proposals/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
I think we can safely delete this file. It is created by default by django but in our case we're not using it.